### PR TITLE
Bump client version to 1.0.11

### PR DESCRIPTION
**Pull request by paulgb**

### CODE REVIEW

**Review by rolyatmax — APPROVED** [review]

File: js-pkg/tests/package.json [hunk 1]
```diff
@@ -10,7 +10,7 @@
   "license": "ISC",
   "dependencies": {
     "bun-types": "^0.5.7",
-    "driftdb": "1.0.10"
+    "driftdb": "*"
```
Comment (rolyatmax) [inline js-pkg/tests/package.json:13]: Does this run the tests against the driftdb that's been published to npm instead of the local one?

Comment (paulgb) [reply js-pkg/tests/package.json:13]: the opposite; it runs tests against the version in the repo

Comment (paulgb) [reply js-pkg/tests/package.json:13]: well, I guess it depends what's in `package-lock.json`? but in this case it's running against the version in the repo I think

Comment (rolyatmax) [reply js-pkg/tests/package.json:13]: Interesting - it must be that npm workspaces overrides this and makes it point to the local version?

Comment (rolyatmax) [reply js-pkg/tests/package.json:13]: Or maybe it only overrides it when the version matches the one in the local repo 🤔 

I just tried making local changes, and they showed up in the test output. Then I changed the driftdb version to some old version and reran `npm i` from the root of the repo and it looked like it was running the old driftdb version, not the local one.

Anyway, was just curious

**Pull request merged**